src/test/java/teammates/test/cases/action/InstructorFeedbackDeleteActionTest.java

```diff
@@ -72,6 +72,9 @@ protected void testAccessControl() throws Exception {
         verifyUnaccessibleWithoutModifySessionPrivilege(submissionParams);
         verifyAccessibleForInstructorsOfTheSameCourse(submissionParams);
 
+        //delete the previous entity
+        FeedbackSessionsLogic.inst().deleteFeedbackSessionCascade(fs.getSessionName(), fs.getCourseId());
+
```

Comment:
Rather than deleting the previous entity, why  not just remove the next part on recreating the same entity? Since nothing about fs was actually modified.